### PR TITLE
Enable "logoutSessionsOnSensitiveChanges" for LB 2.x projects

### PR DESCRIPTION
**Pull request by bajtos**
### Description

- test cleanup: extract bootSandboxWithOptions()
- Modify "empty-server" template to enable
 "logoutSessionsOnSensitiveChanges" for projects using on LoopBack 2.x

#### Related issues

- https://github.com/strongloop/loopback/issues/3068
- requires https://github.com/strongloop/loopback/pull/3109 to be landed and released first

### Checklist

- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by bajtos** [comment]
@slnode test please

File: test/end-to-end.js [hunk 1]
```diff
@@ -562,6 +552,45 @@ describe('end-to-end', function() {
...
+            User.login(CREDENTIALS, next);
+          },
+          function changeEmail(next) {
+            user.updateAttribute('email', 'new@example.com', next);
+          },
+          function verify(next) {
+            app.models.AccessToken.find(function(err, list) {
+              if (err) return next(err);
+              list = list.map(function(t) { return t.toObject(); });
+              expect(list).to.eql([]);
```
Comment (superkhau) [inline test/end-to-end.js]: `expect(list).to.be.empty;`

Comment (bajtos) [inline test/end-to-end.js]: https://github.com/moll/js-must#beware-of-libraries-that-assert-on-property-access

I could add `dirty-chai` and use `.to.be.empty()`, but I feel it's not worth the effort now.

**Comment by bajtos** [comment]
@slnode ok to test

**Comment by bajtos** [comment]
@slnode test please

**Comment by priyavivek1** [comment]
@bajtos : i have loopback 2.22.1 in my app. still i m getting the warnign message.

"The user model "User" is attached to an application that does not specify
whether other sessions should be invalidated when a password or
an email has changed. Session invalidation is important for security
reasons as it allows users to recover from various account breach
situations.

We recommend turning this feature on by setting
"logoutSessionsOnSensitiveChanges" to true in
server/config.json (unless you have implemented your own solution
for token invalidation).

We also recommend enabling "injectOptionsFromRemoteContext" in
User's settings (typically via common/models/*.json file).
This setting is required for the invalidation algorithm to keep
the current session valid"

Its was not coming a week back. but i was try publishing lopback 2.36.0 to our private repo. Seems some dependency carrying this message, Any idea??

**Comment by bajtos** [comment]
@priyavivek1 please run `npm ls` in your project and look for `loopback` instances in the output. You may want to run `npm update` on your local machine first (or `rm -rf node_modules && npm install`), to ensure you have the latest version of all your project dependencies as configured/allowed by `package.json`

**Pull request merged**